### PR TITLE
update Python User Guide and Scala User Guide with supported platforms info

### PR DESCRIPTION
**Pull request by helenlly**
update python.md and scala.md with supported platforms info below: 

- Supported Platforms: Linux and macOS.<br> _**Note:** Windows is not supported._

### CODE REVIEW

**Comment by helenlly** [comment]
test demo :
https://testhelendoc.readthedocs.io/en/latest/doc/UserGuide/python.html
https://testhelendoc.readthedocs.io/en/latest/doc/UserGuide/scala.html

File: docs/readthedocs/source/doc/UserGuide/python.md [hunk 1]
```diff
@@ -25,6 +25,8 @@
   export PATH=$PATH:$JAVA_HOME/bin
   java -version  # Verify the version of JDK.
   ```
+- Supported Platforms: Linux and macOS.<br>
+  _**Note:** Windows is not supported._
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/UserGuide/python.md:29]: Move to before section 1.

File: docs/readthedocs/source/doc/UserGuide/scala.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ You can download the BigDL official releases and nightly build from the [Release
 export SPARK_HOME=folder path where you extract the Spark package
 export BIGDL_HOME=folder path where you extract the BigDL package
 ```
+Supported Platforms: Linux and macOS.<br>
+ _**Note:** Windows is not supported._
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/UserGuide/scala.md:16]: Move to before section 1.

File: docs/readthedocs/source/doc/UserGuide/python.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
 # Python User Guide
 
 ---
+Supported Platforms: Linux and macOS.<br>
+ _**Note:** Windows is not supported._
```
Comment (hkvision) [inline docs/readthedocs/source/doc/UserGuide/python.md:5]: currently not supported

Comment (hkvision) [reply docs/readthedocs/source/doc/UserGuide/python.md:5]: Weird to separate into two lines.
Change to a single line?
```
Supported Platforms: Linux and macOS. Note that Windows is not currently supported.
```

Comment (helenlly) [reply docs/readthedocs/source/doc/UserGuide/python.md]: done

File: docs/readthedocs/source/doc/UserGuide/scala.md (hunk 2 above)
Comment (helenlly) [reply docs/readthedocs/source/doc/UserGuide/scala.md]: done

File: docs/readthedocs/source/doc/UserGuide/python.md (hunk 1 above)
Comment (helenlly) [reply docs/readthedocs/source/doc/UserGuide/python.md]: done

**Comment by hkvision** [comment]
Shall we also add in index.html?

**Review by hkvision — APPROVED** [review]

**Pull request merged**